src/style.rs

```diff
@@ -433,6 +436,7 @@ impl FlexboxLayout {
         margin: Rect::UNDEFINED,
         padding: Rect::UNDEFINED,
         border: Rect::UNDEFINED,
+        gap: Size { width: Dimension::Points(0.0), height: Dimension::Points(0.0) },
```

Comment:
Should the default gap be `Size::UNDEFINED`?

Reply:
Hmm... yes, I suppose it probably should, although it doesn't make much difference as `UNDEFINED` is going go resolve to 0 anyway. It'll make the code look nicer though :)